### PR TITLE
Add antemortem to SSB

### PR DESCRIPTION
**Pull request by CheeseMuffin**

### CODE REVIEW

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -66,6 +66,34 @@ exports.BattleMovedex = {
...
+	// antemortem
+	postmortem: {
+		acc: 85,
+		basePower: 110,
+		category: "Special",
+		shortDesc: "Makes contact. 50% boost Spa, Spe.",
+		id: "postmortem",
+		isNonstandard: true,
+		name: "Postmortem",
+		pp: 10,
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:78]: PP is supposed to be 10, so we need to add `noPPBoosts: true,`

File: mods/seasonal/random-teams.js [hunk 2]
```diff
@@ -19,6 +19,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: "Pilfer",
 				evs: {hp:252, atk: 252, spe:4}, nature: 'Adamant',
 			},
+			'antemortem': {
+				species: 'Clefable', ability:'Multiscale', item: 'Leftovers', gender: 'M', //ask gender
```
Comment (HoeenCoder) [inline mods/seasonal/random-teams.js]: looks like ante's ability should be sheer force or multiscale, also missing a space after the `:`

File: mods/seasonal/random-teams.js [hunk 3]
```diff
@@ -19,6 +19,12 @@ class RandomStaffBrosTeams extends RandomTeams {
...
+				moves: ['Earth Power', 'Cosmic Power', 'Recover', 'Giga Drain'],
+				signatureMove: 'Postmortem',
+				evs: {hp: 252, spa: 252, def: 4}, nature: 'Modest',
```
Comment (HoeenCoder) [inline mods/seasonal/random-teams.js]: nit-picky but def evs before spa evs

**Review by HoeenCoder — COMMENTED** [review]
Sorry for the really late review

File: mods/seasonal/moves.js (hunk 1 above)
Comment (CheeseMuffin) [reply mods/seasonal/moves.js:78]: Done but didn't show up here for some reason

**Pull request merged**